### DISCUSSION TITLE
how increase Flux speed on rtx 3060 12gb

### DISCUSSION THREAD

**Discussion by rafiislambd** [lllyasviel/stable-diffusion-webui-forge#984] [Q&A]
one image generate time 1 minute 20 second
![0](https://github.com/user-attachments/assets/db670645-9739-4a6d-a9a9-fec877d08a6d)
![1](https://github.com/user-attachments/assets/0afc2340-69ad-4ccb-804d-fc4ec295532c)

**Comment by Jonseed** [comment]
That's about the same speed I'm getting on my 3060 12GB. Sometimes it gets down to about 1 min 10 sec for that resolution. I hear this is pretty amazing speed for Flux. For some it is taking a half hour for one image.

**Comment by wisdombox44** [comment]
i am getting simmilar results 10 to 20 seconds up an downs avarage 1:30 1:40 sometimes over 2 min.

**Comment by pflky** [comment]
This current speedup with NF4 is already a 10-20 times speed up. Honestly it's a fair speed, and way faster than the previous FP8/FP16 checkpoints. I mean, keep in mind, Shchnell took well over a minute at 4 steps on even a 4070ti, with lots of memory overrunning, which was a large reason for the slowdown. Now it's about 30 seconds at 20 steps with Flux Dev NF4 at 1024x1024 on a 4070ti, which is comparable to previous SDXL speeds on A1111, with generally better quality and prompt adherence than SDXL. This is cutting edge and I doubt it will get much faster any time soon, as NF4 is already pushing the limits.

**Comment by agustincaniglia** [comment]
How can I run FLUX on forge with an RTX 2060? I don't know where to put the models, when tried in the obvious places, I get error. I uninstalled xformers just in case btw.

**Reply by wzwow** [reply]
civitai search for flux

**Reply by agustincaniglia** [reply]
Yes I go civitai every day. I just don't know how to run FLUX on Forge.

**Comment by wzwow** [comment]
im running on 2060... is 5mins at 20steps > .<
gonna stick to sdxl for awhile...

**Comment by SillySilk** [comment]
Just did a fresh install and still no joy, hoping its still being tweaked

**Comment by 1ragincajun** [comment]
I'm getting about that on my RTX 3060 (12GB). I'm running Automatic (fp16 LoRA)
